File: G1_PPAI/G1_PPA1_E1/Entidades/RespuestaDeCliente.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace G1_PPA1_E1.Entidades
{
    public class RespuestaDeCliente
    {
        //Atributos
        private DateTime fechaEncuesta;
        private RespuestaPosible respuestaSeleccionada;

        //Constructor
        public RespuestaDeCliente(DateTime fechaEncuesta, RespuestaPosible respuestaSeleccionada)
        {
            this.fechaEncuesta = fechaEncuesta;
            this.respuestaSeleccionada = respuestaSeleccionada;
        }
        //Metodos
        public string getDescripcionRta()
        {
            return respuestaSeleccionada.getDescripcionRta();
        }
    }
}

File: PPAI_E3/src/main/java/dsi/g1/entidades/RespuestaDeCliente.java
package dsi.g1.entidades;

import java.util.Date;

public class RespuestaDeCliente {
    private Date fechaEncuesta;
    private RespuestaPosible respuestaSeleccionada;

    public RespuestaDeCliente(Date fechaEncuesta, RespuestaPosible respuestaSeleccionada) {
        this.fechaEncuesta = fechaEncuesta;
        this.respuestaSeleccionada = respuestaSeleccionada;
    }

    public RespuestaPosible getRespuestaSeleccionada() {
        return respuestaSeleccionada;
    }
    public String getDescripcionRta(){
        return respuestaSeleccionada.getDescripcion();
    }
}
